packages/react/src/dialog/CloseIcon.tsx

```diff
@@ -0,0 +1,14 @@
+export const CloseIcon = (
```

Comment:
components should be PascalCase and everything else should be `camelCase`

so let's refactor this into a component
```suggestion
export const CloseIcon = () => (
```